android/src/test/java/com/plaid/gson/PlaidJsonConverterTest.kt

```diff
@@ -0,0 +1,128 @@
+package com.plaid.gson
+
+import com.plaid.link.event.LinkEvent
+import com.plaid.link.event.LinkEventMetadata
+import com.plaid.link.event.LinkEventName
+import com.plaid.link.event.LinkEventViewName
+import com.plaid.link.result.LinkAccount
+import com.plaid.link.result.LinkAccountSubtype
+import com.plaid.link.result.LinkAccountVerificationStatus
+import com.plaid.link.result.LinkError
+import com.plaid.link.result.LinkErrorCode
+import com.plaid.link.result.LinkExit
+import com.plaid.link.result.LinkExitMetadata
+import com.plaid.link.result.LinkExitMetadataStatus
+import com.plaid.link.result.LinkInstitution
+import com.plaid.link.result.LinkSuccess
+import com.plaid.link.result.LinkSuccessMetadata
+import io.mockk.MockK
+import io.mockk.every
+import io.mockk.mockk
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+
+class PlaidJsonConverterTest {
+  companion object {
+    const val SUCCESS_JSON = "{\"publicToken\":\"4\",\"metadata\":{\"institution\":{\"id\":\"2\",\"name\":\"Chase\"},\"accounts\":[{\"id\":\"1\",\"name\":\"My Checking\",\"mask\":\"1234\",\"verificationStatus\":\"manually_verified\",\"type\":\"depository\",\"subtype\":\"checking\"}],\"linkSessionId\":\"3\",\"metadataJson\":\"{\\\"json\\\":\\\"test\\\"}\"}}"
```

Comment:
Can we use kotlin triple quotes (""") to add these json without needing to escape?
I would even format them so they're super easy to read.